src/apple/project.rs

```diff
@@ -121,5 +135,17 @@ pub fn gen(
         .with_arg(dest.join("project.yml"))
         .run_and_wait()
         .map_err(Error::XcodegenFailed)?;
+
+    if !ios_pods.is_empty() || !macos_pods.is_empty() {
+        std::env::set_current_dir(&dest)
+            .map_err(|source| Error::DirChangeFailed { path: dest, source })?;
+        bossy::Command::impure_parse("pod install")
```

Comment:
We should also automate the installation of CocoaPods itself over in `deps.rs`. Optimistically the homebrew formula is fine?